README.md

```diff
@@ -1 +1,106 @@
+
+## Pre-requisites
+
+Kubernetes 1.11 or later.
+
+## Overview.
+
+There are number of chaos experiments that can be performed using this github action you can find the experiments under experiment folder. These experiments are:
+
+- **Pod Delete**:  This experiment causes (forced/graceful) pod failure of random replicas of an application deployment. It tests deployment sanity (replica availability & uninterrupted service) and recovery workflows of the application pod. Visit <a href="https://docs.litmuschaos.io/docs/pod-delete"> pod delete docs</a> for more info.
+
+- **Container Kill**: This experiment executes SIGKILL on container of random replicas of an application deployment. It tests the deployment sanity (replica availability & uninterrupted service) and recovery workflows of an application. Visit <a href="https://docs.litmuschaos.io/docs/container-kill"> container kill docs</a> for more info.
+
+- **Node CPU Hog**: This experiment causes CPU resource exhaustion on the Kubernetes node. The experiment aims to verify resiliency of applications whose replicas may be evicted on account on nodes turning unschedulable (Not Ready) due to lack of CPU resources. Visit <a href="https://docs.litmuschaos.io/docs/node-cpu-hog"> node cpu hog docs</a> for more info.
+
+- **Node Memory Hog**: This experiment causes Memory exhaustion on the Kubernetes node. The experiment aims to verify resiliency of applications whose replicas may be evicted on account on nodes turning unschedulable due to lack of Memory resources. Visit <a href="https://docs.litmuschaos.io/docs/node-memory-hog"> node memory hog docs</a> for more info.
+
+- **Pod CPU Hog**: This experiment causes CPU resource consumption on specified application containers by starting one or more md5sum calculation process on the special file /dev/zero. It Can test the application's resilience to potential slowness/unavailability of some replicas due to high CPU load. Visit <a href="https://docs.litmuschaos.io/docs/pod-cpu-hog"> pod cpu hog docs</a> for more info.
+
+- **Pod Memory Hog**: This experiment causes Memory resource consumption on specified application containers by using dd command which will used to consume memory of the application container for certain duration of time. It can test the application's resilience to potential slowness/unavailability of some replicas due to high Memory load. Visit <a href="https://docs.litmuschaos.io/docs/pod-memory-hog"> Pod Memory hog docs</a> for more info.
+
+- **Pod Network Corruption**: Inject network packet corruption into application pod. Visit <a href="https://docs.litmuschaos.io/docs/pod-network-corruption"> pod network corruption docs</a> for more info.
```

Comment:
format of this para seems different from others. Lets maintain a uniform pattern